tensorflow/core/api_def/base_api/api_def_Acosh.pbtxt

```diff
@@ -1,4 +1,11 @@
 op {
   graph_op_name: "Acosh"
   summary: "Computes inverse hyperbolic cosine of x element-wise."
+  description: <<END
+Given an input tensor, the function computes inverse hyperbolic cosine of every element.
+Input range is [1, infinity). It returns `nan` if the input lies outside the range.
+```python
+tensor `x` is [-2, -0.5, 1, 1.2, 200, 10000]
```

Comment:
Please write examples as code, in this case: 

x = [-2, -0.5, 1, 1.2, 200, 10000]